packages/components/src/utils/hds-breakpoints.ts

```diff
@@ -0,0 +1,12 @@
+// todo: understand if we can generate this list reading the correspondin Sass file
+// see: https://sergiocarracedo.es/2020/07/17/sharing-variables-between-scss-and-typescript/
+
+export const hdsBreakpointsValues = {
```

Comment:
Nit: For these kinds of variable names, how about renaming to remove the "s" from "Breakpoints"?

i.e., change to "hdsBreakpointValues"

When a word is used as an adjective like this it's usual to use the singular form instead of plural.

(Any other opinions on variable naming? @hashicorp/hds-engineering @hashicorp/hds-design )

Reply:
Yeah, makes sense. Actually, I can even get rid of `hdsBreakpointsNames` since it's easily computed, so I am going to leave only the main map and use the name `hdsBreakpoints` (bonus, it's also consistent with the Sass map)

Changed in fe577fa3a85f58e711110d5f90eb0ae829c829a5